.travis.yml

```diff
@@ -5,15 +5,24 @@ scala: 2.12.3
 
 jdk: oraclejdk8
 
-script: sbt ++$TRAVIS_SCALA_VERSION samples/compile test
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION samples/compile test
+  - $(npm bin)/tsc samples/*.ts
 
 before_cache:
   # See http://www.scala-sbt.org/0.13/docs/Travis-CI-with-sbt.html
   # Tricks to avoid unnecessary cache updates
   - find $HOME/.sbt -name "*.lock" | xargs rm
   - find $HOME/.ivy2 -name "ivydata-*.properties" | xargs rm
 
+install:
+  - . $HOME/.nvm/nvm.sh
+  - nvm install stable
+  - nvm use stable
+  - npm install typescript
```

Comment:
Please use fixed versions of Node.js and TypeScript. Something like
```
nvm use 8
npm install typescript@2.6.2
```
Otherwise the CI might start failing simply because a new version of Node.js or TypeScript is released.